coral/src/app/features/topics/acl-request/forms/TopicConsumerForm.tsx

```diff
@@ -76,6 +76,14 @@ const TopicConsumerForm = ({
   const hideIpOrPrincipalField =
     aclIpPrincipleType === undefined || clusterInfo === undefined;
 
+  useEffect(() => {
```

Comment:
I think it would be good to have these values as a variable with a meaningful name and maybe even a comment. Otherwise I think it's very hard (at least) to understand why and what is done here 😅

Reply:
I thought this was explicit enough? " If there is no clusterInfo or if clusterInfo says it's an aivenCluster, hide consumergroup field".

But I'm interested in the questions you asked yourself, so that it can be made better ^^

```ts
  const hideConsumerGroupField =
    clusterInfo === undefined || clusterInfo.aivenCluster === "true";
```

Reply:
For sure a comment would be nice though, to explain why we need this useEffect

Reply:
Added comments ^^

Reply:
Ah, this is a value required by the backend. "not applicable" I bleieve iut stands for ^^'

Reply:
I added a comment about it here https://github.com/aiven/klaw/pull/525/commits/e9fe817334ecf18b4616f4ac9789046f05678d7a#diff-872ec9146c914350f8a6521f3a061bc8ec2b080cdad2b6c1bc731a2b2ee3d55aR84

Reply:
awesome, now it's more clear! 
(pls github show my comment 🤞 🤞 🤞 🤞 )